engine/spverrors/unfold_test.go

```diff
@@ -0,0 +1,55 @@
+package spverrors
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/bitcoin-sv/spv-wallet/models"
+	pkgerrors "github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUnfoldError(t *testing.T) {
+	t.Run("unfold single string error", func(t *testing.T) {
+		err := errors.New("test error")
+		unfolded := UnfoldError(err)
+		require.Equal(t, "[*errors.errorString] test error", unfolded)
+	})
+
+	t.Run("unfold single SPVError", func(t *testing.T) {
```

Comment:
Personally I don't like table tests :P 
Because I can "run or debug" single test case. However, convince me 😄 - why should I use table tests 🙈 ?

Reply:
I agree with Krzysiek in this thread :D 
In VSCode it's painful

Reply:
I haven't said that VSCode is painfull 🙈 I like it 😄

Reply:
I ("Chat GPT") rewrote the test file to make it as "test table" (actually a map).
So, from now I don't have ability to debug a single case 😢 
- But I don't argue at this point, because there is almost nothing to debug here. Potentially, I'll raise this issue on "more complex" tests.